src/footer/footer.scss

```diff
@@ -0,0 +1,9 @@
+.footer {
+  background-color: #90fa69;
```

Comment:
Could you extract the colors into variables in a common .scss file and use it everywhere to keep the colors consistent throughout the application?
_The mentioned common .scss file might contain common style classes as well._

Reply:
Done.